e2e-tests/create-cluster_test.go

```diff
@@ -14,15 +14,16 @@ import (
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/rand"
 	kubevirtv1 "kubevirt.io/api/core/v1"
+	infrav1 "sigs.k8s.io/cluster-api-provider-kubevirt/api/v1alpha1"
+	tests "sigs.k8s.io/cluster-api-provider-kubevirt/e2e-tests"
```

Comment:
This will fail when we'll start enforcing `goimports`. It can be modified by running `make sanity`

But we can do it in a later PR.